sdf/final-project/technical.md

```diff
@@ -1,126 +1,111 @@
 # SDF Final Project Rubric - Technical
-
 This is intended to be filled out by staff members of DPI Tech Apprenticeship program.
 
-Date/Time:
-
-Trainee Name:
-
-Project Name:
-
-Reviewer Name:
+- Date/Time:
+- Trainee Name:
+- Project Name:
+- Reviewer Name:
 
 ## Documentation
-[ ] Is the README up to date and clear?
-
-[ ] Does the README.md give users a detailed description of the project?
-
-[ ] Includes title, description, install steps, contribution guidelines and anything else that might be helpful.
-
-[ ] Formatted using Markdown
-
-[ ] May include screenshots or diagrams if applicable
+- [ ] Is the README up to date and clear?
+- [ ] Does the README.md give users a detailed description of the project?
+- [ ] Includes title, description, install steps, contribution guidelines and anything else that might be helpful.
+- [ ] Formatted using Markdown
+- [ ] May include screenshots or diagrams if applicable
```

Comment:
I think the "may" is a bit too open for grading purposes? Maybe we can require the presence of an ERD on the readme, since that would definitely assist other developers in understanding the codebase.

Reply:
yes